### DISCUSSION TITLE
Does `useColumns` in `selectOnly` have no effect whether it is set to true or false?

### DISCUSSION THREAD

**Discussion by ddxl123** [simolus3/drift#2054] [Q&A] [answered]
```
selectOnly( tableA )
    ..join([ innerJoin( tableB, useColumns: false ) ] ) 
```
Is it equivalent to:
```
select( tableA )
    ..join( [ innerJoin( tableB, useColumns: false )] )
and `tableA` itself is also set to "useColumns: false"
```

So, does `useColumns` in `selectOnly` have no effect whether it is set to true or false?

**Comment by simolus3** [comment, marked as answer]
If you had set `useColumns: false` for `tableA` in the second query, they would have been equivalent. This is not possible though, since `select` always adds all the columns of the main table.

The only real difference is that the default for `useColumns` is `true` when the query starts with `select` and it's `false` when the query starts with `selectOnly`.
